semantic_conventions/metrics/process-runtime-jvm-metrics.yaml

```diff
@@ -25,6 +25,18 @@ groups:
           Pool names are generally obtained via
           [MemoryPoolMXBean#getName()](https://docs.oracle.com/en/java/javase/11/docs/api/java.management/java/lang/management/MemoryPoolMXBean.html#getName()).
 
+  - id: attributes.process.runtime.jvm.network
+    type: attribute_group
+    brief: "Describes JVM network IO metric attributes."
+    attributes:
+      - ref: thread.id
+        requirement_level: opt_in
+      - id: network.direction
```

Comment:
Should we extend the general network conventions to include the direction? https://github.com/open-telemetry/semantic-conventions/blob/bef5a68c2f580e48334bfb2a118345808b2e2125/semantic_conventions/trace/general.yaml#L58

Reply:
hmm I think that makes sense. I've added a new attribute in the general network conventions (`connection.direction`) and referenced it from this metric